File: js/src/maplestory/api/common/dto/character/characterDojang.ts
export abstract class CharacterDojangDto {
  public abstract date: Date | null;
  public abstract characterClass: string;
  public abstract worldName: string;
  public abstract dojangBestFloor: number;
  public abstract dateDojangRecord: Date | null;
  public abstract dojangBestTime: number;
}

File: csharp/MapleStory.OpenAPI/Src/Common/DTO/Character/CharacterDojang.cs
namespace MapleStory.OpenAPI.Common.DTO
{
    public abstract class CharacterDojangDTO
    {
        public abstract DateTimeOffset? Date { get; set; }
        public abstract string CharacterClass { get; set; }
        public abstract string WorldName { get; set; }
        public abstract int DojangBestFloor { get; set; }
        public abstract DateTimeOffset? DateDojangRecord { get; set; }
        public abstract int DojangBestTime { get; set; }
    }
}
